### ISSUE TITLE
registry resource explodes

### ISSUE THREAD

**Issue by matti** [kontena/terraform-provider-kontena#22]
```
provider "kontena" {
  url   = "${module.kontena_platform.master_uri_https}"
  token = "${module.kontena_cli.current_token_id}"
}

resource "kontena_external_registry" "gcr_us" {
  grid     = "${module.kontena_platform.name}"
  url      = "https://us.gcr.io/${module.myproject.gcp_project_name}"
  username = "_json_key"
  password = "password"
  email    = "invalid@example.com"
}
```

```
* kontena_external_registry.gcr_us: kontena_external_registry.gcr_us: Request HTTP GET /v1/external_registries/myproject/us.gcr.io error: Get /v1/external_registries/myproject/us.gcr.io: unsupported protocol scheme ""
```

**Comment by SpComb** [comment]
> `Request HTTP GET /v1/external_registries/myproject/us.gcr.io error: Get /v1/external_registries/myproject/us.gcr.io: unsupported protocol scheme ""`

Can you double-check that the `provider "kontena" { url = ... }` isn't actually an empty string?

The provider should probably validate those...

**Comment by SpComb** [comment]
Presumably fixed in https://github.com/kontena/kontena-client-go/pull/11, but still needs a `govendor update` to pull it in.

**Comment by matti** [comment]
cant be empty if its created?

**Comment by SpComb** [comment]
> cant be empty if its created?

Not sure what you mean... but I was able to repro that error using `provider "kontena" { url = "" }`, so I asume that's what's happening.

**Comment by SpComb** [comment]
Assuming this is fixed in  #23
